src/fabric/src/fabric2_fdb.erl

```diff
@@ -1253,6 +1255,21 @@ fdb_to_revinfo(Key, {?CURR_REV_FORMAT, _, _, _, _, _} = Val) ->
         rev_size => RevSize
     };
 
+fdb_to_revinfo(Key, {?CURR_REV_FORMAT, _, _, _, _} = Val) ->
```

Comment:
This looks wrong to me. There shouldn't be two versions of the current winner rev value.